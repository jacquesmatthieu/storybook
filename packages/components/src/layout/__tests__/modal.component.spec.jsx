import React from 'react';
import { shallow } from 'enzyme';

import Modal from '../modal.component';

describe('Modal', () => {
  let props;

  beforeEach(() => {
    props = {
      title: 'Foo',
      open: true,
    };
  });

  const getWrapper = () => shallow(<Modal {...props}>Bar</Modal>);

  it('should render properly', () => {
    // When
    const wrapper = getWrapper();

    // Then
    expect(wrapper).toMatchSnapshot();
  });
});
