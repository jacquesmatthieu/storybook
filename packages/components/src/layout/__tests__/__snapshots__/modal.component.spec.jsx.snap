// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal should render properly 1`] = `
<Portal>
  <modalcomponent__ModalWrapper
    onKeyDown={[Function]}
  >
    <modalcomponent__ModalShade
      onClick={[Function]}
    />
    <modalcomponent__ModalCard>
      <modalcomponent__ModalTitle
        variant="headline"
      >
        Foo
      </modalcomponent__ModalTitle>
      Bar
    </modalcomponent__ModalCard>
  </modalcomponent__ModalWrapper>
</Portal>
`;
